### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by tianyuchen**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "eslint-config-google",
+    "parserOptions": {
+       "ecmaVersion": 6
+     }
```
Comment (gameboy1024) [inline .eslintrc]: Incorrect indent

File: 016-Power_digit_sum.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  for (let i = 0; i < res.length; ++i) {
+    sum += res[i];
+  }
+
+  return sum;
+}
+
+/**
+ * Calculate num multiple by two
+ * @param {Array} num
```
Comment (gameboy1024) [inline 016-Power_digit_sum.js]: Array<number>

Comment (gameboy1024) [inline 016-Power_digit_sum.js]: `Array<number>`

File: 016-Power_digit_sum.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+ */
+function multiple2(num) {
+  const len = num.length;
+  let res = new Array(len);
+  let carry = 0;
+
+  for (let i = 0; i < len; ++i) {
+    let product = num[i] * 2 + carry;
+    carry = Math.floor(product / 10);
+    res[i] = product - carry * 10;
```
Comment (gameboy1024) [inline 016-Power_digit_sum.js]: Modulo

File: .eslintrc (hunk 1 above)
Comment (tianyuchen) [reply .eslintrc]: where is wrong?

**Pull request merged**